webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleViewHandler.java

```diff
@@ -243,8 +243,11 @@ protected void wrappedService(HttpServletRequest request, HttpServletResponse re
 
                 sendBytes(request, response, totalPdfOutput, "application/pdf");
             } else if (extension.equals("zip")) {
+                String schedule = query.remove("schedule");
```

Comment:
Similar question: could we instead parse the JSON here, rather than passing the string all the way down? That feels more correct to me.

Reply:
Fixed, I think